src/speech_to_text_finetune/config.py

```diff
@@ -1,7 +1,25 @@
+import json
+
 import yaml
 from pydantic import BaseModel
 
 
+def set_cv_languages_mapping(languages_path: str):
+    """
+    Args:
+        languages_path (str): json filepath that stores all languages available for finetuning,
+          see hf_utils/get_available_languages_in_cv() for more details.
+    """
+    with open(languages_path) as json_file:
+        languages_name_to_id = json.load(json_file)
+    return languages_name_to_id
+
+
+LANGUAGES_NAME_TO_ID = set_cv_languages_mapping(
+    "example_data/languages_common_voice_17_0.json"
+)
```

Comment:
Like this: 

```suggestion
LANGUAGES_NAME_TO_ID = {
    "Abkhaz": "ab",
    "Acehnese": "ace",
    "Adyghe": "ady",
    "Afrikaans": "af",
    "Amharic": "am",
    "Aragonese": "an",
    "Arabic": "ar",
    "Mapudungun": "arn",
    "Assamese": "as",
    "Asturian": "ast",
    "Azerbaijani": "az",
    "Bashkir": "ba",
    ...
}
```

Reply:
> values that might change across different Common Voice dataset versions.

what changes between versions (i.e. in the 3 latest versions)? 
I imagine that the only change would be the addition of new languages, which is backwards-compatible (new key, no conflicts).

Reply:
I understand what you mean, my point was (and it is not a strong opinion):

> In any case hardcoding values is not ideal.
> I decided to download this file and add it to the repo.

In this case, you are still hardcoding values but the values just happen to be in a separate json file. If/when you update, are you going to keep older files? How would they be used? 
If you are just going to replace the file with a new one or edit the existing one, having the values in a Python dict is just simpler and make sure that it doesn't rely on a relative path (which can go wrong in many ways).
If you think someone is going to check which version is it, put it in a comment above the dict

Reply:
> In this case, you are still hardcoding values but the values just happen to be in a separate json file.

True, but its clear(er) that this is a file specific to a Common Voice version, and by changing a variable here:
```
get_available_languages_in_cv("mozilla-foundation/common_voice_17_0")
```
 you could download a different version easily, instead of having to go to a website, download the file, and move the new values to the dictionary.

Anyway, I ll use the hardcoded values for now as you suggest, since its not such a big deal and I really need to merge this PR. I ll open an issue to think of a better way to handle this